### ISSUE TITLE
Auto import functionality should be available to fix broken module imports

### ISSUE THREAD

**Issue by simeyla** [microsoft/vscode#102458]
There are all kinds of ways to move TS files around in VSCode. There are built in features, there are plugins. They can fall over themselves for all kinds of reasons leaving many imports in a huge mess.

### Existing 'Auto import' (`Ctrl + .`) functionality

The functionality for `Ctrl + .` to create an auto import is invaluable. However it does not work for an existing broken import (without first deleting it). For example this is the current state of one of my files where many imports are broken:

![image](https://user-images.githubusercontent.com/3656278/87382373-a1715680-c54b-11ea-851e-c8e53d87004d.png)

Currently there are three ways to fix this (that I know of): 

* Delete all the imports and re-add one by one using 'Auto import'
* Delete all the imports and select 'Add all missing imports' for the first underlined symbol.
* Manually fix the path (which may be as simple as adding or removing `../`).  (most cumbersome)

I was unable to find a plugin to do this, and in any case it should probably be natively supported alongside `Auto import`.

### Feature request

I want to be able to `Ctrl + .` for a broken symbol **in the `import` declarations** in order to fix it in place. The functionality would be identical to normal `Ctrl + .` behavior, but it would **update the existing reference instead of adding a new one**.

So basically when I see this...

![image](https://user-images.githubusercontent.com/3656278/87382179-2d36b300-c54b-11ea-8e34-39bc0625c14d.png)

And I type `Ctrl + .`...

![image](https://user-images.githubusercontent.com/3656278/87381803-60c50d80-c54a-11ea-8daa-df6ae924a209.png)

I want to see the same `Auto import` menu I would see elsewhere...

![image](https://user-images.githubusercontent.com/3656278/87381703-2f4c4200-c54a-11ea-9c9d-83b37711998e.png)

This would be a HUGE timesaver. I get scared of refactoring due to this issue.

Selecting a path from the menu would update that `import` statement in place. 
An additional 'Fix all broken imports' option would also be very welcome.

**Comment by mjbvz** [comment]
This sounds like a good candidate for an extension. If the feature is popular enough, we could look into integrating it into VS Code
